src/components/JobLogsContainer/JobLogsContainer.js

```diff
@@ -100,7 +101,7 @@ export const JobLogsContainer = props => {
   };
 
   const handleRowClick = (e, row) => {
-    if (row?.progress?.failed) {
+    if (row?.status !== JOB_EXECUTION_STATUSES.COMPLETED) {
```

Comment:
Could you please try to write a test for this. I think we can mock with Sinon the `open` method and check that upon click on rows with statuses other than COMPLETED the window.open is called with proper arguments.